### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by calvincfso** [jbrennan72/JACS_Lab_for_Morphology#6]
Your tutorial was really helpful in understanding how the scanner worked! It would be nice to see some of those screenshots of your work with the scanner on the github repo as well!

**Comment by MorphoFun** [comment]
> Your tutorial was really helpful in understanding how the scanner worked! It would be nice to see some of those screenshots of your work with the scanner on the github repo as well!

I agree! Having the visuals is really helpful to guide users through the different steps.
